modules/__tests__/_bc-isActive-test.js

```diff
@@ -2,12 +2,14 @@ import expect from 'expect'
 import React from 'react'
 import { render, unmountComponentAtNode } from 'react-dom'
 import createHistory from 'history/lib/createMemoryHistory'
+import useRouterHistory from '../useRouterHistory'
 import IndexRoute from '../IndexRoute'
 import Router from '../Router'
 import Route from '../Route'
-import qs from 'qs'
 import shouldWarn from './shouldWarn'
 
+const createRouterHistory = useRouterHistory(createHistory)
+
 describe('v1 isActive', function () {
```

Comment:
I haven't gotten to the `isActive` deprecations yet. It'll go whenever I can get to that.